### ISSUE TITLE
Error on BFRES: FMAT

### ISSUE THREAD

**Issue by Satoshi64** [Syroot/io_scene_bfres#2]
Import script crashes when it reaches FMAT m_face_mat1.
[sample.zip](https://github.com/Syroot/io_scene_bfres/files/788823/pc003_03.zip)
[crash.txt](https://github.com/Syroot/io_scene_bfres/files/788816/crash.txt)

**Comment by RayKoopa** [comment]
This doesn't look like an MK8 BFRES. Other games are not supported at the moment.

**Comment by Satoshi64** [comment]
It is from genei ibunroku fe, extracts fine with BFRESExtractor (by LordNerd i guess) and io_BFRES_vertex is even able to load the model properly (without textures).

**Comment by RayKoopa** [comment]
Yeah that might be, however I'm not actively developing io_scene_bfres anymore so other BFRES file formats might never be added (beyond the fact MK8 support has never been finished).
